sqlair.go

```diff
@@ -389,10 +346,9 @@ func (q *Query) GetAll(sliceArgs ...any) (err error) {
 }
 
 type TX struct {
-	sqltx   *sql.Tx
-	sqlconn *sql.Conn
-	db      *DB
-	done    int32
+	sqltx *sql.Tx
+	db    *DB
```

Comment:
And since it's only used here to _pass_ to `Query`, we can remove it here too.

Reply:
You are right! These were in part used to prevent running the finalizers which is no longer happening.